src/app.ts

```diff
@@ -14,11 +15,11 @@ app.use(express.static("public", {
 }));
 
 app.get(`${urlBase}.jar`, verifyParams, normaljar)
-//Although we don't use `verifyParams` for the pom, it's still good to
-//prevent malformed maven coordinates producing a valid pom
+// Although we don't use `verifyParams` for the pom, it's still good to
+// prevent malformed maven coordinates producing a valid pom
 app.get(`${urlBase}.pom`, verifyParams, pom)
-app.get(`${urlBase}.md5`, (_, res) => res.sendStatus(404))
-app.get(`${urlBase}.sha1`, (_, res) => res.sendStatus(404))
+app.get(`${urlBase}.pom.:algorithm`, verifyParams, pomHash)
+app.get(`${urlBase}.jar.:algorithm`, verifyParams, jarHash)
```

Comment:
Alright, im a little confused on this one still.. Would you like me to add back in the two removed lines? 
ie:
```
app.get(`${urlBase}.md5`, (_, res) => res.sendStatus(404))
app.get(`${urlBase}.sha1`, (_, res) => res.sendStatus(404))
```
Maybe im misunderstanding, but I can't think of a scenario where in `/cuse/maven/jei-238222/2724420/jei-238222-2724420.md5` would be called either, I would expect it to include a file type like `.jar.md5` or as you mentioned `.zip.md5`?

Reply:
Oh, and for the bit about Pom/Jar checksums: I think the the jar checksums are very important as ive ran into issues with corrupt jar files being downloaded due to occasional network blips. Also it's a fairly light-weight operation; for the amount of trouble I see a lot of reward. Pom checksums, I see your point. Is there any possibility for caching entire poms/checksums on either a `.pom` or `.pom.:algorithm` route call on fs or in memory? It would fix our issues of slow pom (and checksum) generation and having a checksum available is just generally good practice, though I can see how its not entirely needed.

Reply:
Caching the POM isn't possible, as cursemaven runs in a serverless environment. There is no persistent storage (unless you add a database or the like, which I won't do).

Reply:
Ah, alright so no pom caching. Ill remove the checksums in a new commit and then rePR when I get a chance.